petastorm/unischema.py

```diff
@@ -197,10 +201,6 @@ def __str__(self):
     def fields(self):
         return self._fields
 
-    @property
```

Comment:
Shouldn't you add a property _name

Reply:
I was trying to do that before, so I changed the name of the `_name` attribute to `__name`. That resulted in inability to depickle the unischema in existing datasets. It was a mistake to use pickling to store Unischema as pickle from the start. It makes it much harder to change the code without breaking backward compatibility. For now, leaving _name an attribute (which is still accessible by users if really needed). The only downside is that a user may assign to `_name`. I would think this is not a very big deal though...